physical_properties/pKa/analysis/analysis_of_typeIII_predictions/README.md

```diff
@@ -7,7 +7,7 @@ One-to-one matching of predicted pKas to experimental pKas was performed with tw
 1. **Closest**: Each predicted pKa is matched to experimental pKa values that minimize the absolute error of that pair.
 When more than one predicted pKa match to the same experimental pKa, only the predicted pka that has the lowest absolute error is kept. 
 Predicted pKas that were not matched to experimental pKas were excluded from this analysis.
-2. **Hungarian**: Experimental pKas and predicted pKas are matched following [Hungarian algorithm](https://docs.scipy.org/doc/scipy-0.18.1/reference/generated/scipy.optimize.linear_sum_assignment.html) which aims to minimize the global cost of assigning the one-to-one correspondences between two sets. It finds the optimum assignment between experimental and predicted set of pKas that minimizes linear sum of squared errors of all pairwise matches. Cost is defined as squared error instead of absolute error to minimize the effect of extra predicted pKas to overall matching. We acknowledge Kiril Lanevskij for suggesting this alternative assignment of experimental pKas.
+2. **Hungarian**: Experimental pKas and predicted pKas are matched following Hungarian algorithm. It finds the optimum assignment between experimental and predicted set of pKas that minimizes linear sum of squared errors of all pairwise matches. Cost is defined as squared error instead of absolute error to minimize the effect of extra predicted pKas to overall matching.
```

Comment:
I think you should keep a link to some description of a Hungarian algorithm, I understand not using the scipy docs if you're not actually using scipy here, but most people won't be familiar with this pairing so it would be nice to link to somewhere they can read about it, even if its wikipedia. 

Also, there was some grammatical errors that I tried to fix, I would use this instead:
```
Experimental pKas and predicted pKas are matched following the Hungarian algorithm. It finds the optimum pairing of experimental and predicted pKas to minimize the linear sum of squared errors for each molecule. We chose to minimize the squared error instead of absolute error to minimize the effect of extra predicted pKas to overall matching.
```
I removed the reference to "Cost" because I felt it could confuse the reader, but I'm also I a little confused about my absolute error would change how extra predicted pKas affect the matching? Could you explain that in the comments here or in the text itself?

As a note - I don't know if "the Hungarian algorithm" or if "a Hungarian algorithm" makes more sense, but it needs an article.

Reply:
As far as I understand, if you use squared errors instead of absolute error the value of cost function is exaggerated for matches of extra predicted pKas ( the ones that have extremely low or high values). So minimizing squared errors favors local matching of closer values more than minimizing absolute error.

Reply:
I incorporated other changes in this comment.